### PR TITLE
Backend impl error handling

### PR DESCRIPTION
**Pull request by Ionis-Kutrolli**

### CODE REVIEW

**Comment by Ionis-Kutrolli** [comment]
I don't know why it shows so many files as changed since a few of them were changed in a pr that was already merged

File: src/main/java/com/google/impactdashboard/servlets/ErrorHandlerServlet.java [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+   */
+  @Override
+  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    Throwable exception = (Throwable) request.getAttribute("javax.servlet.error.exception");
+    String message = (String) request.getAttribute("javax.servlet.error.message");
+
+    ErrorMessage error = new ErrorMessage(message, exception);
+
+    Gson gson = new Gson();
+    String json  = gson.toJson(error);
```
Comment (peterpie123) [inline src/main/java/com/google/impactdashboard/servlets/ErrorHandlerServlet.java:29]: nit: two spaces after `json`

File: src/main/webapp/WEB-INF/web.xml [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+    </servlet>
+    <servlet-mapping>
+        <servlet-name>ErrorHandler</servlet-name>
+        <url-pattern>/error-handler</url-pattern>
+    </servlet-mapping>
+    <error-page>
+        <exception-type>java.lang.RuntimeException</exception-type>
+        <location>/error-handler</location>
+    </error-page>
+</web-app>
```
Comment (peterpie123) [inline src/main/webapp/WEB-INF/web.xml:14]: nit: no newline at EOF

File: src/main/java/com/google/impactdashboard/servlets/ErrorHandlerServlet.java (hunk 1 above)
Comment (Ionis-Kutrolli) [reply src/main/java/com/google/impactdashboard/servlets/ErrorHandlerServlet.java:29]: I don't know how you saw that but good catch

Comment (peterpie123) [reply src/main/java/com/google/impactdashboard/servlets/ErrorHandlerServlet.java]: Me neither!

**Pull request merged**